docs/developers/testing.rst

```diff
@@ -0,0 +1,110 @@
+==================
+Run MicroOVN tests
+==================
+
+MicroOVN has two types of tests, linter checks and functional tests and this
+page will show how to run them.
+
+Linter checks
```

Comment:
Run linter checks (?)

Reply:
I'm not sure I understand this. Imo the two headers, `Linter checks` and `Functional tests`, make sense here as their sections describe different types of test that user might want to run